msg.py

```diff
@@ -72,7 +72,7 @@
         '{1}.',
         '{1}.',
         'Ohh, {1}. Ich mag die Zahl, {0}! ;)',
-        '{1}?! Was ist das denn für eine Zahl. Zähl mal bis {3}, das ist viel geiler!',
+        '{1}?! Was ist das denn für eine Zahl? Zähl mal bis {3}, das ist viel geiler!',
```

Comment:
I disagree with this particular line change.

(And to be honest, I kinda want to force you to understand how to undo small changes.)